engine/getstarted/linux_install_help.md

```diff
@@ -3,36 +3,49 @@ description: Getting started with Docker
 redirect_from:
 - /mac/started/
-title: 'Example: Install Docker on Ubuntu Linux'
+title: Install Docker using a shell script
 ---
 
-This installation procedure for users who are unfamiliar with package
-managers, and just want to try out the Getting Started tutorial while running Docker on Linux. If you are comfortable with package managers, prefer not to use
-`curl`, or have problems installing and want to troubleshoot, please use our
-`apt` and `yum` <a href="/engine/installation/"
-target="_blank">repositories instead for your installation</a>.
+This installation procedure for users who don't want to use a package manager to
+install Docker. If you can use a package manager, you should use the
+[recommended installation procedure for your operating system](/engine/installation/)
+instead. Using a package manager ensures that you get upgrades when they are
+available, and allows you to install a specific version of Docker, rather than
+the very latest version.
 
-1. Log into your Ubuntu installation as a user with `sudo` privileges.
+## Prerequisites
 
-2. Verify that you have `curl` installed.
+- You need `sudo` access on Linux, or administrator access on Windows.
+- You need `curl` or `wget` installed. These instructions use `curl`, but you
+  can adapt them to use `wget`.
 
-        $ which curl
+## Install Docker using the `install.sh` script
 
-    If `curl` isn't installed, install it after updating your manager:
+> **Warning**: Always examine shell scripts you download from the internet before
+> running them.
 
-        $ sudo apt-get update
-        $ sudo apt-get install curl
+1.  Open [https://get.docker.com](https://get.docker.com/) in your web browser
+    so that you can examine the script before running it. This is important
+    because the script will run with elevated privileges.
 
-3. Get the latest Docker package.
+2.  Run the script, using `curl` to download it and piping it through `sh`:
 
-        $ curl -fsSL https://get.docker.com/ | sh
+    ```bash
+    $ curl -faSL https://get.docker.com/ | sh
+    ```
 
-    The system prompts you for your `sudo` password. Then, it downloads and
-    installs Docker and its dependencies.
+    You are prompted for your `sudo` password. The script determines your
+    operating system, downloads Docker and its dependencies, and installs Docker.
 
-    >**Note**: If your company is behind a filtering proxy, you may find that the
-    >`apt-key`
-    >command fails for the Docker repo during installation. To work around this,
-    >add the key directly using the following:
+    > **Note**: If your company is behind a filtering proxy, the
+    > `apt-key` may fail during Docker installation. To work around this,
+    > use the following command to manually add the Docker key:
     >
     >       $ curl -fsSL https://get.docker.com/gpg | sudo apt-key add -
+
+  3.  The script attempts to start Docker and to configure the system to start
+      Docker automatically.
```

Comment:
Only on systems using "apt" (`.deb`) packages, on RPM based platforms, docker is not started, or configured to start automatically (per convention for RPM based installs)